docker/build-tools/Dockerfile

```diff
@@ -302,6 +324,8 @@ COPY --from=ruby_tools_context /usr/local/bin /usr/local/bin
 COPY --from=ruby_tools_context /usr/local/lib /usr/local/lib
 COPY --from=nodejs_tools_context /usr/local /usr/local
 COPY --from=nodejs_tools_context /node_modules/ /node_modules/
+COPY --from=python_context /usr/local/bin /usr/local/bin
```

Comment:
this should be /usr/lib - copying /usr/local/bin will leave the python3.5 runtime behind...

Reply:
and /usr/bin :)